tailwind.config.js

```diff
@@ -8,11 +8,11 @@ module.exports = {
   ],
   theme: {
     screens: {
-      sm: '900px',
-      md: '1100px',
-      lg: '1300px',
-      xl: '1440px',
-      '2xl': '1700px',
+      sm: '414px',
+      md: '834px',
+      lg: '874px',
+      xl: '946px',
+      '2xl': '1440px',
```

Comment:
@muhadovic why do you need those changes?

Reply:
I think we have to imagine the small screens. But the sm in the current screen is 900px, it is so large value. Also, I think it is good to keep the tailwindcss default values. I don't understand why anyone set the large screen values.

Reply:
@muhadovic we agreed to screen sizes. We are all open for any suggestion. If you suggest to change the screen sizes, you should get approval from everyone. Ask everyone over Slack.

Reply:
@muhadovic one more thing to note, you should consider this app is running on desktop, not mobile.

Reply:
@dbizhev  Yes, That is the desktop version. But we have to imagine when the users resize the app.